lambdas/accesstoken/src/main/java/uk/gov/di/ipv/cri/passport/accesstoken/validation/TokenRequestValidator.java

```diff
@@ -66,6 +78,24 @@ private void validateMaxAllowedAuthClientTtl(JWTAuthenticationClaimsSet claimsSe
         }
     }
 
+    private void validateJwtId(JWTAuthenticationClaimsSet claimsSet) throws InvalidClientException {
+        JWTID jwtId = claimsSet.getJWTID();
+        if (jwtId == null || StringUtils.isBlank(jwtId.getValue())) {
+            LOGGER.error("The client auth JWT id (jti) is missing");
+            throw new InvalidClientException("The client auth JWT id (jti) is missing");
+        }
+        ClientAuthJwtIdItem clientAuthJwtIdItem =
+                clientAuthJwtIdService.getClientAuthJwtIdItem(jwtId.getValue());
+        if (clientAuthJwtIdItem != null) {
+            LoggingUtils.appendKey("jti", jwtId.getValue());
+            LoggingUtils.appendKey("usedAtDateTime", clientAuthJwtIdItem.getUsedAtDateTime());
```

Comment:
Might be nice to make these keys constants

Reply:
Added to `LogField` enum